### PR TITLE
🚨 refactor(examples): Home

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/903

# Scope of work
refactor homepage example

# Screenshots of designs
mobile:
image

desktop:
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [ ] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

File: packages/vue/src/examples/pages/home/Home.vue [hunk 1]
```diff
@@ -209,157 +209,269 @@ export default {
...
           rating: { max: 5, score: 4 },
-          isOnWishlist: false
+          isOnWishlist: false,
+          reviews: 8,
+          badgeLabel: "",
+          badgeColor: "color-primary"
         }
-      ]
+      ],
+      sectionLevel: 3,
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/home/Home.vue:290]: you can set heading level directly on SfHeading component ;)

**Comment by pspaczek** [comment]
@AdamPawlinski and please remove unused .jpg and .png :)

File: packages/vue/src/examples/pages/home/Home.vue [hunk 2]
```diff
@@ -209,157 +209,269 @@ export default {
...
+    order: 1;
+    margin: var(--spacer-xl) 0 0 0;
   }
-}
-.banner-central {
-  @include for-desktop {
-    --banner-container-flex: 0 0 70%;
+  &__heading {
+    display: flex;
+    order: 2;
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/home/Home.vue:357]: I think we don't need flex for whole page and order properties;)

File: packages/vue/src/examples/pages/home/Home.vue [hunk 3]
```diff
@@ -209,157 +209,269 @@ export default {
...
-    margin: 0;
-    & + & {
-      margin: 0 0 0 calc(var(--spacer-big) / 2);
-      @include for-desktop {
-        margin: 0 0 0 var(--spacer-big);
-      }
+  &__instagram-feed {
+    --image-width: 100%;
+    --image-height: 100%;
+    display: grid;
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/home/Home.vue:391]: are you test it on IE?

File: packages/vue/src/examples/pages/home/Home.vue (hunk 2 above)
Comment (AdamPawlinski) [reply packages/vue/src/examples/pages/home/Home.vue:357]: @pspaczek  should I use sth else to change order of components between mobile and desktop? or the order should be the same (from desktop or mobile)?

File: packages/vue/src/examples/pages/home/Home.vue (hunk 3 above)
Comment (AdamPawlinski) [reply packages/vue/src/examples/pages/home/Home.vue]: If you mean edge it looks fine 
image, but I found another bug:  
image

**Comment by pspaczek** [comment]
not: we should do the fix for product-cards shadows on the carousel

**Pull request merged**